app/uk/gov/hmrc/apiplatformjobs/scheduled/DeleteUnregisteredDevelopersJob.scala

```diff
@@ -73,4 +76,11 @@ class DeleteUnregisteredDevelopersJobLockService @Inject() (repository: LockRepo
   override val ttl: duration.Duration         = 1.hours
 }
 
-case class DeleteUnregisteredDevelopersJobConfig(initialDelay: FiniteDuration, interval: FiniteDuration, enabled: Boolean, limit: Int)
+case class DeleteUnregisteredDevelopersJobConfig(initialDelay: FiniteDuration, interval: FiniteDuration, enabled: Boolean, limit: Int) {
+
+  // These emails are used for unregistered users in QA acceptance testing, so are excluded from deletion
```

Comment:
not a fan of these being hardcoded inside of production code. In theory they will not be deleted on any application in any env,